src/pages/trade/ui/chart.tsx

```diff
@@ -7,11 +7,102 @@ import { useCandles } from '../api/candles';
 import { observer } from 'mobx-react-lite';
 import { DurationWindow, durationWindows } from '@/shared/utils/duration.ts';
 
-const ChartLoadingState = () => {
+export const ChartLoadingState = () => {
```

Comment:
suggestion: could you make this fade animate (sort of like the summary component) so it's not as jarring?

Reply:
does it need the `export`?

Reply:
Removed the export, and added a fade-in animation.